auth/routes/oauth_routes.go

```diff
@@ -6,10 +6,12 @@ const (
 	OAuthBegin    = "OAuthBegin"
 	OAuthCallback = "OAuthCallback"
 	LogOut        = "LogOut"
+	PostCustom    = "PostCustom"
 )
 
 var OAuthRoutes = rata.Routes{
 	{Path: "/auth/logout", Method: "GET", Name: LogOut},
 	{Path: "/auth/:provider/callback", Method: "GET", Name: OAuthCallback},
 	{Path: "/auth/:provider", Method: "GET", Name: OAuthBegin},
+	{Path: "/auth/:provider/", Method: "POST", Name: PostCustom},
```

Comment:
@Chumper, can you explain why this route needs to be added. This seems that it would affect any oauth provider, not just GIthub.